other/autoregressive-process/utils.jl

```diff
@@ -0,0 +1,42 @@
+# Generates an AR(p) process with coefficients `ϕ`.
+# `ϕ` should be provided as a vector and it represents the coefficients of the AR model.
+# Hence the order of the generated process is equal to the length of `ϕ`.
+# `s` indicates the total length of the series to be generated.
+function generate_process(ϕ::AbstractVector{Float32}, s::Int)
+    @assert s > 0 "s must be positive"
+    # Generate white noise
+    ϵ = randn(Float32, s)
+    # Initialize time series
+    X = zeros(Float32, s)
+    p = length(ϕ)
+    X[1] = ϵ[1]
+    # Reverse the order of the coefficients for multiplication later on
+    reverse!(ϕ) 
```

Comment:
```suggestion
    ϕ = reverse(ϕ) 
```
For safety. The overhead of copying a 3-vector should be negligible.